From 5879d03e7a6c96dd4804bccf95c7d28f89ed7525 Mon Sep 17 00:00:00 2001
From: Gilles Dubreuil <gilles@redhat.com>
Date: Fri, 12 Dec 2014 15:40:58 +1100
Subject: [PATCH] JSON was invalid

Created an warning/error message when applying Puppet

$ cat metadata.json | json_verify
lexical error: inside a string, '\' occurs before a character which it
may not.
                                        {   "name": "puppet-galera",
                     (right here) ------^
---
 galera/metadata.json | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/galera/metadata.json b/galera/metadata.json
index 636b8a4..26a8cb5 100644
--- a/galera/metadata.json
+++ b/galera/metadata.json
@@ -1,7 +1,7 @@
 {
   "name": "puppet-galera",
   "version": "0.0.2",
-  "author": "Ryan O\'Hara",
+  "author": "Ryan O'Hara",
   "summary": "Install/Configure MariaDB with galera",
   "license": "Apache 2.0",
   "source": "git://github.com/redhat-openstack/puppet-galera.git",
@@ -30,7 +30,7 @@
         "6",
         "7"
       ]
-    },
+    }
   ],
   "requirements": [
     {
